vql/MFT.yaml

```diff
@@ -19,6 +19,9 @@ parameters:
    - name: DateBefore
      type: timestamp
      description: "search for events before this date. YYYY-MM-DDTmm:hh:ssZ"
+   - name: AllDrives
```

Comment:
This change is just a parameter. Need to also add the Alldrives switch in the artifact itself.

Reply:
just added this in. Remember we need to modify the template file to change the artifact.